setup.py

```diff
@@ -80,16 +80,27 @@
 if has_cython:
     cmdclass['build_ext'] = build_ext
 
+runtime_lib_dirs =  os.environ['TA_LIBRARY_PATH']
```

Comment:
I think ``os.environ['FOO']`` raises a ``KeyError`` if not found, so this won't work.  Instead, this might (using ``os.environ.get`` to provide a default value if key is not found:

```python
runtime_lib_dirs = os.environ.get('TA_LIBRARY_PATH', '').split(os.pathsep)
```